doc/release-notes.md

```diff
@@ -28,22 +35,378 @@ reindex or re-sync the whole chain.
+be signalling. If a sufficient number of blocks mined signal for activation (starting at 80% and ticking down towards 60%)
+by the end of a cycle, the hard fork will change status from started to locked_in. At this point, hard fork activation
+will occur once another cycle ends. 
+
+Now that the hard fork is active, the new rules will be in effect.
+
+Masternode Reward Location Reallocation
+---------------------------------------
+
+Once the MN_RR hard fork activates, some coinbase funds will be moved into the Credit Pool balance (to Platform).
+Evonodes will then receive a single reward per payment cycle on the Core chain - not quad rewards (4 sequential blocks)
+as in v19/v20. The remainder of evonode payments will be distributed by Platform from the asset lock pool. This is to
+incentivize evonodes to upgrade to platform, because only nodes running platform will get these migrated rewards.
+
+
+Mainnet Spork Hardening
+-----------------------
+
+This version hardens all Sporks on mainnet. Sporks remain in effect on all devnets and testnet; however, on mainnet,
+the value of all sporks are hard coded to 0, or 1 for SPORK_21_QUORUM_ALL_CONNECTED. These hardened values match the
+active values historically used on mainnet, so there is no change in the network's functionality.
+
+Default Branch Changed
+----------------------
+
+The Dash Core repository (`github.com/dashpay/dash`) will be using `develop` as its default branch. New clones
+of the repository will no longer default to the `master` branch. If you want to continue using a stable version of
+Dash Core you should manually checkout the `master` branch.
+
+# Additional Changes
+
+Wallet
+------
+
+### Avoid Partial Spends
+The wallet will now avoid partial spends (APS) by default, if this does not result
+in a difference in fees compared to the non-APS variant. The allowed fee threshold
+can be adjusted using the new `-maxapsfee` configuration option. (dash#5930)
+
+
+### Experimental Descriptor Wallets
+
+Please note that Descriptor Wallets are still experimental and not all expected functionality
+is available. Additionally there may be some bugs and current functions may change in the future.
+Bugs and missing functionality can be reported to the [issue tracker](https://github.com/dashpay/dash/issues).
+
+v21 introduces a new experimental type of wallet - Descriptor Wallets. Descriptor Wallets store
+scriptPubKey information using descriptors. This is in contrast to the Legacy Wallet
+structure where keys are used to generate scriptPubKeys and addresses. Because of this
+shift to being script based instead of key based, many of the confusing things that Legacy
+Wallets do are not possible with Descriptor Wallets. Descriptor Wallets use a definition
+of "mine" for scripts which is simpler and more intuitive than that used by Legacy Wallets.
+Descriptor Wallets also uses different semantics for watch-only things and imports.
```

Comment:
```suggestion
v21 introduces a new experimental type of wallet - Descriptor Wallets. Descriptor Wallets store
scriptPubKey information using descriptors. This is in contrast to the Legacy Wallet
structure where keys are used to generate scriptPubKeys and addresses. Because of this
shift to being script-based instead of key-based, many of the confusing things that Legacy
Wallets do are not possible with Descriptor Wallets. Descriptor Wallets use a definition
of "mine" for scripts which is simpler and more intuitive than that used by Legacy Wallets.
Descriptor Wallets also uses different semantics for watch-only things and imports.
```